.upstream-tests/test/cuda/test_platform.pass.cpp

```diff
@@ -91,6 +91,15 @@ __host__ __device__ void test() {
       printf("Host success\r\n");,
       printf("Device success\r\n");
   )
+
+  NV_DISPATCH_TARGET(
+    NV_IS_HOST, (;),
```

Comment:
Clever, but maybe just use `NV_IF_TARGET`?

Reply:
I'd wanted to make sure that empty parens don't break things, the semicolon was an artifact, but I am leaving this in.